scss/docs/_patterns_icons.scss

```diff
@@ -46,3 +46,4 @@
 // so include them here for the purposes of
 // presenting the application icons on the docs page.
 @include vf-p-icons--application;
+@include vf-p-icons--additional;
```

Comment:
As we haven't released the `vf-p-icons--application` I'm tempted to remove it and allow only adding individual icons.

There is no way we can be sure what given product will need.

We could have `vf-p-icons--all` in case someone want them all, but I don't think we should encourage that, as the CSS will be huge.

Reply:
Fair point - in that case, how/where should I document them? Should they still have their own example?

Reply:
I think we should put all icons that are not included by default (so previous 'application' and 'additional' together in one example).

I'm not sure about putting them on the docs page each in separate little "card", as this will be a huge list. But on the other hand that seems to be the best way to quickly find what you need and grab class name without looking into source code.

Reply:
Cool - I've updated so that there is no longer an application set. I haven't created an `vf-p-icons--all` set, I thought if it's something we'd rather people didn't do, then it's best not to give them the ability. What do you think?

Reply:
Yes, I agree that we shouldn't encourage it. Even if it would make our case simpler when we want to include them all for examples.